script/cross-compile

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Usage: script/cross-compile <version>
+#
+# Packages the project over a matrix of supported OS and architectures and
+# prints the asset filenames and labels suitable for upload.
+set -e
+
+version="${1?}"
+
+echo '
+  darwin   amd64    OS X
+  freebsd  386      FreeBSD 32-bit
+  freebsd  amd64    FreeBSD 64-bit
+  linux    386      Linux 32-bit
+  linux    amd64    Linux 64-bit
+  windows  386      Windows 32-bit
+  windows  amd64    Windows 64-bit
```

Comment:
I'll test but I don't think the recent gopass bump will fix the issue, since it only contained fixes for Windows. I have no idea why it doesn't compile.